examples/list-publications-template.html

```diff
@@ -62,7 +62,7 @@
             );
         </script>
 
-        <!--   Sample event listener on Incito offer -->
+        <!--   Sample event listener on Incito click -->
         <script>
             window.addEventListener(
                 'tjek-publication-list-item-clicked',
```

Comment:
Do we still emit events on window?

Reply:
Yes, this is the old implementation. Will refactor it to use the container element :)